DESCRIPTION

```diff
@@ -15,9 +15,11 @@ BugReports: https://github.com/tidyverse/purrr/issues
 Depends: 
     R (>= 4.0)
 Imports:
+    carrier (>= 0.1.1),
```

Comment:
For now, lets keep carrier and mirai as suggests. That will make the syntax a little less appealing, but purrr is such a widely used package I want to be extra careful before adding any new packages.

Reply:
Yes, it does complicate the syntax a little bit as we can't re-export `daemons()` and `crate()`, but it's fine to start there.